metta/common/formula/ModusPonensFormula.metta

```diff
@@ -0,0 +1,14 @@
+;; Formula for the modus ponens rule
+
+;; Import modules
+!(import! &self ../truthvalue/TruthValue.metta)
+!(import! &self ../Num.metta)
+
+(= (modus-ponens-formula (STV $Ps $Pc)
+                         (STV $PQs $PQc))
+   (let*
+       (($notPQs 0.2)
+        ($notPQc 1))
```

Comment:
@leungmanhin, could you add a comment about this temporary hack?

You can probably more or less copy/paste the following

https://github.com/opencog/pln/blob/08c100f9b9c26f69dd38e1934c70fd2c9e70e316/opencog/pln/rules/propositional/modus-ponens.scm#L16-L31